api/mesh/v1alpha1/dataplane_insight_helpers.go

```diff
@@ -44,7 +44,7 @@ func (ds *DataplaneInsight) UpdateCert(generation time.Time, expiration time.Tim
 		return err
 	}
 	ds.MTLS.CertificateExpirationTime = ts
-	ds.MTLS.CertificateRegenerations++
+	ds.MTLS.CertificateRegenerations += uint32(certs)
```

Comment:
If certificateRegeneration process happens for all certificates at a time, do we really want to count every certificate separately?

Reply:
Yes, because we need to know how many certs were generated